aiobotocore/awsrequest.py

```diff
@@ -28,3 +28,31 @@ async def _text_prop(self):
     @property
     def text(self):
         return self._text_prop()
+
+
+class HttpxAWSResponse(AWSResponse):
```

Comment:
can we have this inherit from `AioAWSResponse`, that way you just have to replace `_content_prop`

Reply:
trying to figure out a way to mask that `aread` function, that was a terrible decision on the part of `httpx` IMO.

Reply:
actually we can just remove the class entirely with a little bit of logic in `AioAWSResponse`